src/Common/src/System/Security/Cryptography/Asn1Reader/AsnReader.Integer.cs

```diff
@@ -107,9 +107,7 @@ public BigInteger ReadInteger(Asn1Tag expectedTag)
             }
             finally
             {
-                // Clear the whole tmp so that not even the sign bit is returned to the array pool.
-                Array.Clear(tmp, 0, tmp.Length);
-                ArrayPool<byte>.Shared.Return(tmp);
+                CryptoPool.Return(tmp, CryptoPool.ClearAll);
```

Comment:
Same ClearAll question.  Seems like either we should be explicit at all call sites, in which case we should make the parameter non-optional, or we should remove the argument when it's the default.